radix-engine-toolkit/examples/transactions/account-derive-from-mnemonic/Go/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"crypto/ed25519"
+	"encoding/hex"
+	"fmt"
+	"strconv"
+
+	"github.com/anyproto/go-slip10"
+	"github.com/radixdlt/radix-engine-toolkit-go/v2/radix_engine_toolkit_uniffi"
+	"github.com/tyler-smith/go-bip39"
+)
+
+func assert(condition bool, message string) {
+    if !condition {
+        panic(message)
+    }
+}
+
+// networkId: 1 - Mainnet, 2 - Stokenet, ...
+// mnemonic: words list space separated
+// passphrase: optional
```

Comment:
What does optional mean? Empty string? Sorry, no prior Rust experience.

Reply:
I've updated that comment to: `passphrase: for Wallet compatible account use empty string`